qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -273,6 +273,25 @@ def convert_set_frequency(self, shift, instruction):
         }
         return self._qobj_model(**command_dict)
 
+    @bind_instruction(instructions.ShiftFrequency)
+    def convert_shift_frequency(self, shift, instruction):
+        """Return converted `ShiftFrequency`.
+
+        Args:
+            shift (int): Offset time.
+            instruction (ShiftFrequency): Shift frequency instruction.
+
+        Returns:
+            dict: Dictionary of required parameters.
+        """
+        command_dict = {
+            'name': 'shiftf',
+            't0': shift+instruction.start_time,
+            'ch': instruction.channel.name,
+            'frequency': instruction.frequency
```

Comment:
Are we missing a factor of 1e9 here? What are the expected units of shiftf on the backend?

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/4390/commits/7ecf5ab6be4f22c58b889b1969bfba31b34a4545.